src/pages/api/feedback.ts

```diff
@@ -0,0 +1,100 @@
+import type { NextApiRequest, NextApiResponse } from "next";
+import { google } from "googleapis";
+import requestIp from "request-ip";
+import { UAParser } from "ua-parser-js";
+
+export default async function feedbackFormHandler(
+  request: NextApiRequest,
+  response: NextApiResponse
+) {
+  const { method } = request;
+  if (method !== "POST") {
+    return response
+      .status(405)
+      .send({ message: "only POST requests are allowed" });
+  }
+
+  // Values to pass to spreadsheet
+  const body = request.body;
+  const type = body.category;
```

Comment:
Nitpick: you can destructure this as `const { type, comment: feedbackText } = body;`. The `comment: feedbackText` is getting the property from `body` and also assigning it to a new variable `feedbackText`. This is not needed but just showing another way to do this.